src/components/JoinUsBar/JoinUsBar.js

```diff
@@ -0,0 +1,57 @@
+import { opacify } from "polished";
+import React, { Component } from "react";
+import { Link } from 'react-router-dom'
+import { setOpacity } from "react"
+import "./JoinUsBar.css";
+
+/*Attributo "color" per visualizzare la barra rossa (red) o blu (blue)*/
+/*Default: red*/
+
+/*Sistemare opacità*/
+
+const blue = {
+  backgroundColor: "#120068"
+}
+
+const red = {
+  backgroundColor: "#fc3f1a"
+}
+
+const white = {
+  backgroundColor: "#ffffff"
+}
+
+function handleColor_bar(color) {
+  var style_bar = red;
+  if(color == "blue"){
+    style_bar=blue;
+  }
+  return style_bar;
+}
+
+function handleColor_button(color) {
+  var style_button = white;
+  if(color == "blue"){
+    style_button=red;
+  }
+  return style_button;
+}
+
+export default class JoinUsBar extends Component {
+  render() {
+    console.log("Il colore della barra JoinUs è:", this.props.color)
+    return (
+        <div className="join_us_bar" style ={handleColor_bar(this.props.color)} >
```

Comment:
Questo non è male, ma forse sarebbe meglio creare due classi separate, una per lo sfondo blu e una per lo sfondo rosso e tramite una condizione (if) sul props applicare una o l'altra.

Ex:
file.css:

```
.bg-blue {
    background: var(--blue);
}

.bg-red {
    background: var(--red);
}
```

linea javascript:
```
<div className=`join-us-bar ${this.props.color == "red" ? bg-red : bg-blue}`  >
```

Per quanto riguarda il js ci sono 2 concetti importanti: 
il primo è quello di variabili inserite all'interno di una stringa, chiamato template literals (che riguarda gli apici al contrario e il dollaro con relative parentesi, e puoi trovare docs qui:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?retiredLocale=it

il secondo è quello di condizionale (if) inline (ternary operator)
docs:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator